app/views/image_templates/edit.html.erb

```diff
@@ -8,6 +8,11 @@
   <hr>
 </div>
 
+<% @image_template.match_options['threshold'] ||= 80 %>
```

Comment:
Switching to the argument pass caused the tests to fail, with the default value not being detected anymore. I've switched it back to explicit assignment in the template and am inclined to keep it there unless there is a security issue with it.